src/transform/update_let.rs

```diff
@@ -28,10 +29,10 @@ impl crate::Transform for UpdateLet {
     fn transform(
         &self,
         relation: &mut RelationExpr,
-        _: &HashMap<GlobalId, Vec<Vec<ScalarExpr>>>,
+        state: TransformArgs,
```

Comment:
I'm not 100% behind `state` as the argument name here. It wouldn't be the worst name we have, but I figure `state` is the CS version of `stuff`.